### PR TITLE
feat: Add SSL policy to LB

### PR DESCRIPTION
**Pull request by zacharyblasczyk**

### CODE REVIEW

**Review by cavila-evoliq — APPROVED** [review]
Looks good!

File: modules/app_lb/https/main.tf [hunk 1]
```diff
@@ -10,21 +10,29 @@ resource "google_compute_managed_ssl_certificate" "default" {
...
+  name       = "${var.namespace}-https"
   target     = google_compute_target_https_proxy.default.id
   port_range = "443"
   ip_address = var.ip_address
+  labels     = var.labels
+}
 
-  labels = var.labels
+# SSL Policy to apply to Target Https Proxy
+resource "google_compute_ssl_policy" "ssl-policy" {
```
Comment (jsbroks) [inline modules/app_lb/https/main.tf:30]:
```suggestion
resource "google_compute_ssl_policy" "default" {
```

File: modules/app_lb/https/main.tf [hunk 2]
```diff
@@ -10,21 +10,29 @@ resource "google_compute_managed_ssl_certificate" "default" {
...
 resource "google_compute_target_https_proxy" "default" {
-  name    = "${var.namespace}-https-proxy"
-  url_map = var.url_map.id
-  ssl_certificates = [
-    google_compute_managed_ssl_certificate.default.id
-  ]
+  name             = "${var.namespace}-https-proxy"
+  url_map          = var.url_map.id
+  ssl_certificates = [google_compute_managed_ssl_certificate.default.id]
+  ssl_policy       = google_compute_ssl_policy.ssl-policy.id
```
Comment (cavila-evoliq) [inline modules/app_lb/https/main.tf:16]: Line 16 will need to be changed to `ssl_policy = google_compute_ssl_policy.default.id` as a consequence of the resource being renamed on line 30

**Review by jsbroks — APPROVED** [review]
nj!

**Review by zythosec — APPROVED** [review]

**Comment by jsbroks** [comment]
This PR is included in version 1.14.0 :tada:

**Pull request merged**